include/myHashPartitionerCb.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef CVDP_MY_HASH_PARTITIONER_CB_H
```

Comment:
Is this file actually used?

Reply:
The only place this file is used is in the `myHashPartitionerCb.cpp` implementation file. Should we remove this?

Reply:
If we're not using it let's remove both the header and code files for it, yes.

Reply:
These have been removed.